charts/pega/charts/installer/templates/_helpers.tpl

```diff
@@ -54,6 +56,8 @@
 {{- end }}
 
 {{- define "waitForRollingUpdates" -}}
+{{- $deploymentName := printf "%s-" (include "installerDeploymentName" $) -}}
+{{- $deploymentNameRegex := printf "%s- " (include "installerDeploymentName" $) -}}
```

Comment:
Should this also include the `-` as it is used where `pega-` was previously used?

Reply:
Discussed privately - the dash gets added elsewhere as part of the deployment names, and the key feature of `deploymentNameRegex` is that it includes the extra space.